hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/s3guard/ITestS3GuardToolLocal.java

```diff
@@ -97,7 +98,6 @@ public void testImportCommand() throws Exception {
         .getListing().size());
     assertEquals("Expected 2 items: empty directory and a parent directory", 2,
         ms.listChildren(parent).getListing().size());
-    assertTrue(children.isAuthoritative());
```

Comment:
why did you cut this assertion?

Reply:
This looks wrong assertion. No -authoritative flag is given in the cmd.

Reply:
Actually I got assertion error when I ran the test with fs.s3a.metadatastore.impl=org.apache.hadoop.fs.s3a.s3guard.LocalMetadataStore.
```
$ mvn verify -Ds3guard -Dtest=x -Dit.test=ITestS3GuardToolLocal
...
[ERROR] Tests run: 33, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 208.832 s <<< FAILURE! - in org.apache.hadoop.fs.s3a.s3guard.ITestS3GuardToolLocal
[ERROR] testImportCommand(org.apache.hadoop.fs.s3a.s3guard.ITestS3GuardToolLocal)  Time elapsed: 15.872 s  <<< FAILURE!
java.lang.AssertionError
	at org.apache.hadoop.fs.s3a.s3guard.ITestS3GuardToolLocal.testImportCommand(ITestS3GuardToolLocal.java:101)
```

Reply:
probably is wrong then :)